File: CSharp/WinFormsLeeson3/task2/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace task2
{
    class Product
    {
        public string Name { get; set; }
        public string Specification { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public override string ToString() => $"{Name} {Specification}";
    }
}

File: Java/ConsoleProjects/Shop/src/Shop/Models/Product.java
package Shop.Models;

public class Product {
    private String name;
    private Double price;

    public Product(String name, Double price) {
        if(name != null && price != null) {
            this.name = name;
            this.price = price;
        }
        else throw new NullPointerException();
    }
    public String getName() {
        return name;
    }

    public Double getPrice() {
        return price;
    }


    @Override
    public String toString() {
        return name + "(" + price + " грн)";
    }
}